docs/onboarding/cli.md

```diff
@@ -101,34 +104,34 @@ config:get enableMetrics
 Starts a miner and subscribes to new blocks for the node. The node has to be synced with the network for the miner to start mining.
 
 ```sh
-ironfish miners:start
+ironfish miners:start  
 ```
 
-Join a mining pool.
```

Comment:
Can you keep the Join a mining pool? so 
Join a mining pool with the number of CPU threads to utilize for mining (-t) and setting other than your default account address for block rewards (-a)

`ironfish miners:start` is joining a mining pool. `ironfish miners:pools` is starting your own mining pool.